src/jit/lsra.cpp

```diff
@@ -4897,7 +4924,9 @@ LinearScan::allocateBusyReg(Interval *current, RefPosition *refPosition)
         if (assignedInterval == nullptr)
         {
             RefPosition* nextPhysRegPosition = physRegRecord->getNextRefPosition();
-            assert(nextPhysRegPosition->nodeLocation == refLocation && candidateBit != candidates);
+
+            // nodeLocation is not always == refLocation, so this assert is removed
+            //assert(nextPhysRegPosition->nodeLocation == refLocation && candidateBit != candidates);
             continue;
```

Comment:
As far as I recall ARM64 doesn't support partial structs passed in register and on the stack. I think this is only for ARM32.